sample-apps/react/egress-composite/src/components/layouts/Spotlight/Spotlight.scss

```
@@ -1,13 +1,15 @@
+    flex-wrap: wrap;
+    height: 100vh;
+    align-items: center;
+    align-content: center;
+    justify-content: center;
 
-  .str-video__participant-view {
-    transition: width 0.2s ease-in-out;
-    max-width: unset;
+    .str-video__participant-view {
+      transition: width 0.2s ease-in-out;
+      // max-width: unset;
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)